### PR TITLE
ACP-4189: Payone Paypal Express - Show Express Checkout Buttons on Cart Page.

### PR DESCRIPTION
**Pull request by abitskil**
Ticket: https://spryker.atlassian.net/browse/ACP-4189

### CODE REVIEW

File: src/SprykerEco/Yves/PaypalPaymentApp/Theme/default/components/molecules/paypal-buttons/paypal-buttons.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
+/* tslint:disable: no-any */
+declare const paypal: any;
+/* tslint:enable */
+
+import Component from 'ShopUi/models/component';
+import ScriptLoader from 'ShopUi/components/molecules/script-loader/script-loader';
+import AjaxLoader from 'ShopUi/components/molecules/ajax-loader/ajax-loader';
```
Comment (matweew) [inline src/SprykerEco/Yves/PaypalPaymentApp/Theme/default/components/molecules/paypal-buttons/paypal-buttons.ts:7]: add ShopUI to composer.json dependencies.

File: LICENSE [hunk 2]
```diff
@@ -1,28 +1,21 @@
...
-
-Upon termination of the License as provided under this Agreement, Licensee shall promptly destroy the Software and any back-up copy of the Software made during the Evaluation Period or Early Access Period if Spryker and Licensee have not agreed a non-evaluation license to Spryker’s software.
-
-This Agreement shall be governed by the laws of Germany to the exclusion of IPR (International Law) and the United Nations Convention on Contracts for the International Sale of Goods (CISG). The parties consent to the jurisdiction of the courts in Berlin (Germany).
-
-This Agreement and/or the License is not assignable or transferable by Licensee and any attempt to do so is null and void.
-
-This Agreement constitutes the entire agreement between the parties concerning Licensee’s use of the Software. This Agreement supersedes any prior verbal understanding between the parties. This Agreement may be amended only in a writing signed by an authorized officer of Spryker.
-
+The MIT License (MIT)
```
Comment (matweew) [inline LICENSE:1]: are you sure?

Comment (abitskil) [reply LICENSE:1]: Yes, by default the ECO modules are distributed under the MIT license.

File: src/SprykerEco/Shared/PaypalPaymentApp/Transfer/paypal_payment_app.transfer.xml [hunk 3]
```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
+<transfers xmlns="spryker:transfer-01"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="spryker:transfer-01 http://static.spryker.com/transfer-01.xsd">
+
+    <transfer name="PaypalQueryParameters" strict="true">
```
Comment (pushokwhite) [inline src/SprykerEco/Shared/PaypalPaymentApp/Transfer/paypal_payment_app.transfer.xml:6]: Transfer module has to be 3.27.0 at least in composer dependencies.

**Review by pushokwhite — APPROVED** [review]

**Pull request merged**